apps/tangle-dapp/lib/hyperlane/transfer.ts

```diff
@@ -11,6 +11,7 @@ export async function hyperlaneTransfer(params?: {
   recipientAddress: string;
   token: BridgeTokenType;
   amount: string;
+  ethersProvider: any;
```

Comment:
Consider leaving a TODO to give this a type later on to avoid possible future type system / unexpected bugs caused by `any`